### ISSUE TITLE
About the training code of nnUNet in the pre-training stage

### ISSUE THREAD

**Issue by wangshansong1** [Luffy03/Large-Scale-Medical#17]
Excellent work! Thanks for your open source spirit, it saves me a lot of time!

Could you provide the training code for nnUNet in the pre-training phase? Similar to "single_train.sh" and "voco_train.py", I found that the current ssl framework can only be used to train swinunetr. Or, is it because I didn't look carefully enough? Which code can implement the above steps? Can you provide a simple tutorial?

Another question, I found that the code saves the model every 20,000 steps instead of directly verifying it. Can you provide the code to find the optimal model on the validation set? Or, I guess overfitting is not important, and the model of the last step can be directly used for downstream tasks? Because I found this phenomenon on my own large-scale dataset.

Best.

**Comment by Luffy03** [comment]
Dear shansong, thank you very much for your attention to our work.
1. We did not provide the codes for nnunet SSL training, but you can modify [Self-supervised/models/PlainConvUNet_load.py](https://github.com/Luffy03/Large-Scale-Medical/blob/main/Self-supervised/models/PlainConvUNet_load.py) for it.
2. Usually validation is not very critical step in SSL and we simply use the model of the last step.

**Comment by wangshansong1** [comment]
> Dear shansong, thank you very much for your attention to our work.
> 
> 1. We did not provide the codes for nnunet SSL training, but you can modify [Self-supervised/models/PlainConvUNet_load.py](https://github.com/Luffy03/Large-Scale-Medical/blob/main/Self-supervised/models/PlainConvUNet_load.py) for it.
> 2. Usually validation is not very critical step in SSL and we simply use the model of the last step.

Thanks for the quick answer! I guess that's it, my own experiments came to the same conclusion!
